ecosystem/sep-0031.md

```diff
@@ -437,6 +437,8 @@ Name | Type | Description
 `receiver_id` | `string` | (optional) The ID included in the [SEP-12 PUT /customer](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#customer-put) response for the Receiving Client. Required if the Receiving Anchor requires SEP-12 KYC on the Receiving Client.
 `fields` | object | An object containing the values requested by the Receiving Anchor in the `GET /info` endpoint.
 `lang` | string | (optional) Defaults to `en`. Language code specified using [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1).  Any human-readable error codes or field descriptions will be returned in this language.
+`refund_memo` | (optional) The memo the Receiving Anchor must use when sending refund payments back to the Sending Anchor. If not specified, the Receiving Anchor should use the same memo the Sending Anchor used to send the original payment.
+`refund_memo_type` | (optional) The type of the `refund_memo`. Can be `id`, `text`, or `hash`. See the [memos](https://developers.stellar.org/docs/encyclopedia/memos) documentation for more information.
```

Comment:
🤔  Should anchors use a default value for the memo type? I mean, if the user sends a memo but no memo type, should it be considered a text memo?

Reply:
If we do want to standardize a default type, it should probably be `id`. That way the ecosystem could migrate to using muxed account addresses for refunds eventually.

But I don't know if providing a default makes sense. Maybe I should just add a clause to the definitions saying `refund_memo` and `refund_memo_type` must be sent together.

Reply:
> But I don't know if providing a default makes sense. Maybe I should just add a clause to the definitions saying `refund_memo` and `refund_memo_type` must be sent together.

Yeah, that makes sense to me.